pkg/clusteragent/autoscaling/workload/telemetry.go

```diff
@@ -86,17 +88,31 @@ var (
 		"Tracks the changes in autoscaler conditions",
 		telemetry.Options{NoDoubleUnderscoreSep: true},
 	)
+
+	// telemetryLocalFallbackEnabled tracks whether local fallback recommendations are being used
+	telemetryLocalFallbackEnabled = telemetry.NewGaugeWithOpts(
+		subsystem,
+		"local_fallback_enabled",
+		[]string{"namespace", "target_name", "autoscaler_name", le.JoinLeaderLabel},
+		"Tracks whether local fallback recommendations are being used",
+		commonOpts,
+	)
 )
 
 func trackPodAutoscalerStatus(podAutoscaler *datadoghq.DatadogPodAutoscaler) {
 	for _, condition := range podAutoscaler.Status.Conditions {
 		if condition.Status == corev1.ConditionTrue {
 			autoscalingStatusConditions.Set(1.0, podAutoscaler.Namespace, podAutoscaler.Name, string(condition.Type), le.JoinLeaderValue)
+		} else {
```

Comment:
i think this `else` got removed during a merge - this metric hasn't been reporting any values as a result